Tools/scripts/param_check.py

```diff
@@ -80,7 +80,7 @@ def parse_arguments():
     return args
 
 
-def check_file(file, metadata, skip : SkippedChecks = None):
+def check_file(file, metadata, skip : SkippedChecks | None = None):
```

Comment:
I think you need to make an import to have `| None` work on all Python versions we support

```
from __future__ import annotations
```